packages/app/src/client/integrations/common/UninstallIntegrationBtn.tsx

```diff
@@ -55,23 +57,31 @@ export const UninstallBtn = ({
         tenant_id: tenant.id,
         id: integration.id
       });
-      try {
-        await deleteFolder({ integration, tenant });
-      } catch (error) {
-        registerNotification({
-          state: "error" as const,
-          title: "Could not delete grafana folder",
-          information: error.response.data.message ?? error.message
-        });
-      }
+      await deleteFolder({ integration, tenant });
       dispatch(deleteIntegration({ tenantId: tenant.id, id: integration.id }));
       history.push(installedIntegrationsPath({ tenant }));
     } catch (error) {
-      registerNotification({
-        state: "error" as const,
-        title: "Could not uninstall integration",
-        information: error.response.errors?.[0].message ?? error.message
-      });
+      let notification: Parameters<typeof registerNotification>[0]
+      if (isGraphQLClientError(error)) {
+        notification = {
+          state: "error" as const,
+          title: "Could not uninstall integration",
+          information: error.response.errors![0].message
```

Comment:
needs the `!` because the `ClientError` is not very precisely typed. 
Added https://github.com/prisma-labs/graphql-request/pull/282 upstream to fix that, let's see what they say.